src/app/lib/config/services/persian.js

```diff
@@ -203,7 +203,7 @@ export const service = {
     radioSchedule: {
       hasRadioSchedule: true,
       onFrontPage: true,
-      frontPagePosition: 'top-stories',
+      frontPagePosition: '',
```

Comment:
Since Persian radio schedules are removed, should the `hasRadioSchedule` and `onFrontPage` values be updated to be `false`?

Reply:
`onFrontPage` has been set to `false` but `hasRadioSchedule` can be left as `true` for now as we might need radio schedule on other page types.